src/components/utils/src/file_system.cc

```diff
@@ -100,7 +100,7 @@ uint32_t file_system::FileSize(const std::string &path) {
 
 uint32_t file_system::DirectorySize(const std::string& path) {
 #ifdef OS_WIN32
-	return 1024 * 1024;
+  return 1024 * 1024:
```

Comment:
- Please change this line code with end of semicolon ";"
- Please add necessary comment why here directory size should be 1024*1024